OpenXmlPowerTools/HtmlToWmlCssParser.cs

```diff
@@ -835,13 +833,11 @@ public Color ToColor()
             }
             else
             {
-                try
+                Color c;
+                if (ColorParser.TryFromName(m_value, out c))
```

Comment:
`out var c`

Reply:
It's a C# 7.0 feature, the project is currently on C# 4.0 (https://github.com/OfficeDev/Open-Xml-PowerTools/blob/vNext/OpenXmlPowerTools/OpenXmlPowerTools.csproj.DotSettings) Do you confirm I should change this ? I'm totally ok with that but wanted to be sure :)

Reply:
Yup. I didn't realize it was still on 4.0! Go ahead and update it.